pkgs/development/python-modules/djangorestframework-simplejwt/default.nix

```
@@ -12,15 +12,15 @@
 
 buildPythonPackage rec {
   pname = "djangorestframework-simplejwt";
-  version = "5.3.0";
+  version = "5.3.1";
   format = "setuptools";
 
   disabled = pythonOlder "3.7";
```

```suggestion
  disabled = pythonOlder "3.8";
```
https://github.com/jazzband/djangorestframework-simplejwt/blob/8ae34dd128ef8c5a79d4632e29fa82162bc9a787/CHANGELOG.md?plain=1#L4